src/content/docs/logs/log-management/ui-data/drop-data-drop-filter-rules.mdx

```diff
@@ -11,79 +11,90 @@ redirects:
-* You reduce noise by removing irrelevant events and attributes.
+* Lower costs by storing only logs relevant to your account.
+* Protect privacy and security by removing personal identifiable information (PII).
+* Reduce noise by removing irrelevant events and attributes.
 
-## How drop filter rules work
+<Callout variant="caution">
+  Use caution when deciding to drop data. The data you drop is not recoverable. Before using this feature, review the [responsibilities and considerations for dropping data](#caution).
+</Callout>
 
-A drop filter rule matches data based on a query. When triggered, the drop filter rule removes the matching data from the ingestion pipeline before it is written to in NRDB. Since the data does not reach the backend, it cannot be queried: the data is gone and cannot be restored.
+## How drop filter rules work [#how-it-works]
+
+A drop filter rule matches data based on a query. When triggered, the drop filter rule removes the matching data from the ingestion pipeline before it is written to NRDB.
+
+This creates an explicit demarcation between the logs being forwarded from your domain and the data that New Relic collects. Since the data removed by the drop filter rule does not reach our backend, it cannot be queried: the data is gone and cannot be restored.
 
 ![New Relic Logs - Architecture - Drop filter](./images/NRDB2.png "New Relic Logs - Architecture - Drop filter")
 
 <figcaption>
   During the ingestion process, log data can be parsed, transformed, or dropped before being stored.
 </figcaption>
 
-## Create drop filter rules [#logs-in-context-retention]
+## Cautions when dropping data [#caution]
+
+When creating drop rules, you are responsible for ensuring that the rules accurately identify and discard the data that meets the conditions that you have established. You are also responsible for monitoring the rule, as well as the data you disclose to New Relic.
 
-<Callout variant="caution">
-  You must have [admin permissions](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#user-group) in New Relic to create and edit drop filters, or be a member of a role with create and edit permissions for **Logging Parsing Rules**.
-</Callout>
+New Relic cannot guarantee that this functionality will completely resolve data disclosure concerns you may have. New Relic does not review or monitor how effective the rules you develop are.
 
-Drop filter rules can be created from **[one.newrelic.com](http://one.newrelic.com) > Logs** using new or existing [log queries](/docs/logs/new-relic-logs/ui-data/query-syntax-logs).
+Creating rules about sensitive data can leak information about what kinds of data you maintain, including the format of your data or systems (for example, through referencing email addresses or specific credit card numbers). Rules you create, including all information in those rules, can be viewed and edited by any user with the relevant role-based access control permissions.
```

Comment:
"Any user with relevant (...) permissions can view the rules you create, including..."